diff --git a/node_modules/@meteora-ag/dlmm/dist/index.d.ts b/node_modules/@meteora-ag/dlmm/dist/index.d.ts
index c7da384..a040393 100644
--- a/node_modules/@meteora-ag/dlmm/dist/index.d.ts
+++ b/node_modules/@meteora-ag/dlmm/dist/index.d.ts
@@ -1,5 +1,5 @@
 import * as _coral_xyz_anchor from '@coral-xyz/anchor';
-import { Program, IdlAccounts, ProgramAccount, BN as BN$1, IdlTypes, EventParser } from '@coral-xyz/anchor';
+import { Program, IdlAccounts, ProgramAccount, IdlTypes, EventParser } from '@coral-xyz/anchor';
 import * as _solana_web3_js from '@solana/web3.js';
 import { PublicKey, Connection, AccountMeta, Keypair, TransactionInstruction, Cluster, AccountInfo, Transaction, GetProgramAccountsFilter } from '@solana/web3.js';
 import Decimal from 'decimal.js';
@@ -7,6 +7,7 @@ import { IdlDiscriminator } from '@coral-xyz/anchor/dist/cjs/idl';
 import { Mint } from '@solana/spl-token';
 import { AllAccountsMap } from '@coral-xyz/anchor/dist/cjs/program/namespace/types';
 import BN from 'bn.js';
+import BN$1 from 'bn.js';
 
 /**
  * Program IDL in camelCase format in order to be used in JS/TS.
diff --git a/node_modules/@meteora-ag/dlmm/dist/index.mjs b/node_modules/@meteora-ag/dlmm/dist/index.mjs
index 0c20638..1069ba6 100644
--- a/node_modules/@meteora-ag/dlmm/dist/index.mjs
+++ b/node_modules/@meteora-ag/dlmm/dist/index.mjs
@@ -1,5 +1,5 @@
 // src/dlmm/index.ts
-import { BN as BN21 } from "@coral-xyz/anchor";
+import BN21 from "bn.js";
 import {
   AccountLayout,
   NATIVE_MINT as NATIVE_MINT2,
@@ -23,7 +23,7 @@ import Decimal9 from "decimal.js";
 
 // src/dlmm/constants/index.ts
 import { LAMPORTS_PER_SOL, PublicKey } from "@solana/web3.js";
-import { BN } from "@coral-xyz/anchor";
+import BN from "bn.js";
 
 // src/dlmm/dlmm.json
 var dlmm_default = {
@@ -9534,7 +9534,8 @@ var DlmmSdkError = class extends Error {
 };
 
 // src/dlmm/helpers/index.ts
-import { AnchorProvider, BN as BN18, Program as Program2 } from "@coral-xyz/anchor";
+import { AnchorProvider, Program as Program2 } from "@coral-xyz/anchor";
+import BN18 from "bn.js";
 import {
   ASSOCIATED_TOKEN_PROGRAM_ID,
   NATIVE_MINT,
@@ -9554,9 +9555,7 @@ import {
 } from "@solana/web3.js";
 
 // src/dlmm/types/index.ts
-import {
-  BN as BN2
-} from "@coral-xyz/anchor";
+import BN2 from "bn.js";
 import {
   PublicKey as PublicKey2
 } from "@solana/web3.js";
@@ -9668,11 +9667,11 @@ var MEMO_PROGRAM_ID = new PublicKey2(
 var REBALANCE_POSITION_PADDING = Array(31).fill(0);
 
 // src/dlmm/helpers/binArray.ts
-import { BN as BN8 } from "@coral-xyz/anchor";
+import BN8 from "bn.js";
 import { PublicKey as PublicKey5 } from "@solana/web3.js";
 
 // src/dlmm/helpers/math.ts
-import { BN as BN7 } from "@coral-xyz/anchor";
+import BN7 from "bn.js";
 import Decimal5 from "decimal.js";
 
 // src/dlmm/helpers/u64xu64_math.ts
@@ -9780,12 +9779,12 @@ function pow(base, exp) {
 }
 
 // src/dlmm/helpers/weight.ts
-import { BN as BN6 } from "@coral-xyz/anchor";
+import BN6 from "bn.js";
 import gaussian from "gaussian";
 import Decimal4 from "decimal.js";
 
 // src/dlmm/helpers/weightToAmounts.ts
-import { BN as BN5 } from "@coral-xyz/anchor";
+import BN5 from "bn.js";
 import Decimal3 from "decimal.js";
 
 // src/dlmm/helpers/token_2022.ts
@@ -11677,7 +11676,7 @@ import BN12 from "bn.js";
 import Decimal6 from "decimal.js";
 
 // src/dlmm/helpers/fee.ts
-import { BN as BN11 } from "@coral-xyz/anchor";
+import BN11 from "bn.js";
 function getBaseFee(binStep, sParameter) {
   return new BN11(sParameter.baseFactor).mul(new BN11(binStep)).mul(new BN11(10)).mul(new BN11(10).pow(new BN11(sParameter.baseFeePowerFactor)));
 }
@@ -13243,7 +13242,7 @@ function getTokenProgramId(lbPairState) {
 }
 
 // src/dlmm/helpers/strategy.ts
-import { BN as BN17 } from "@coral-xyz/anchor";
+import BN17 from "bn.js";
 var DEFAULT_MAX_WEIGHT = 2e3;
 var DEFAULT_MIN_WEIGHT = 200;
 function toWeightSpotBalanced(minBinId, maxBinId) {
